SpeechService/SpeechService.cs

```diff
@@ -38,6 +38,7 @@ private ISoundOut activeSpeech
             }
         }
         private int activeSpeechPriority;
+        private decimal transmitVolume;
```

Comment:
Why add a persistent variable here rather than overriding the volume only when the user has explicitly set an override volume in their command?

Reply:
I've given this some more thought I can see a case for persisting the value until the end of the current speech (particularly if we want to separate VoiceAttack contexts for `say`, `transmit` and `volume` from one another).
I'm still not sure that we should persist the value beyond the current script or VoiceAttack command though.